### DISCUSSION TITLE
【keyball44】一部キーが反応しません

### DISCUSSION THREAD

**Discussion by moko-poi** [Yowkees/keyball#666] [Q&A]
お世話になっております。
keyball44を組み立てているのですが、一つのkeyが反応しません。
- キーソケットが原因かと思い、新品と交換したり、ハンダし直しても解決しませんでした
お手数をおかけしますが、原因について助言をいただきたいです。
![スクリーンショット 2024-10-03 23 39 12](https://github.com/user-attachments/assets/58a431a0-4f82-408e-b6f8-415c2a93e13d)

**Comment by jtuyen** [comment]
Is the installed diode for the key in the correct direction? Post some photos of the MCU, there may be a short on COL0 or ROW2 pins.

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。ご返信が遅くなり申し訳ありませんでした。

下記画像の緑丸部のダイオードの半田付けを再加熱してみてください。
確かにテンプラもしくは、未半田付けのように見えます。

![KB44LEFTdiode_fix](https://github.com/user-attachments/assets/5d24f77f-6812-464b-b9fa-306724f89522)

お手数ですがご対応お願いいたします。
